appveyor.yml

```
@@ -0,0 +1,52 @@
+# DO NOT CHANGE the "init" and "install" sections below
+
+# Download script file from GitHub
+init:
+  ps: |
+        $ErrorActionPreference = "Stop"
+        Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
```

I would either fork this script or use a static link to it:
```suggestion
        Invoke-WebRequest https://raw.githubusercontent.com/krlmlr/r-appveyor/7630f7e5802fee9d458af28c1d6531a1062da48d/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
```

---

Excellent idea, I will include it in the next commit! Thank you very much!